pkg/corerp/frontend/controller/secretstores/types.go

```diff
@@ -19,4 +19,11 @@ package secretstores
 const (
 	// ResourceTypeName is the resource type name for secret stores.
 	ResourceTypeName = "Applications.Core/secretStores"
+
+	// The following are possible required keys in a SecretStore depending on it's SecretType
```

Comment:
nit: let's not use different comment formats within the same file (example above for `ResourceTypeName`).

https://tip.golang.org/doc/comment#var